src/utils/amounts.ts

```diff
@@ -1,8 +1,16 @@
+import { BigNumber } from "ethers";
+import { Token } from "src/domain";
+
 interface TokenAmountToFiatParams {
   amount: number;
   token: "eth" | "dai";
 }
 
+interface BigNumberToNumberParams {
```

Comment:
Didn't we agree to put every params interface just above the function it serves?